llvm/include/llvm/TargetParser/X86TargetParser.def

```diff
@@ -175,8 +175,6 @@ X86_FEATURE_COMPAT(AVX512BF16,      "avx512bf16",            34)
 X86_FEATURE_COMPAT(AVX512VP2INTERSECT, "avx512vp2intersect", 35)
 // Below Features has some missings comparing to gcc, it's because gcc has some
 // not one-to-one mapped in llvm.
-X86_FEATURE_COMPAT(3DNOW,           "3dnow",                  0)
-X86_FEATURE       (3DNOWA,          "3dnowa")
```

Comment:
@FreddyLeaf IIRC, we need to find two features to fill here, right?

Reply:
Yes, if gcc also deleted 3dnow, then find 1 feature to fill here. if not, then find 2.

Reply:
Sorry, I realized even gcc removed, they will keep one blank here to keep backward compatibility. So two features are required.
according to this comment above the file: 
```
// We cannot re-adjust the position of X86_FEATURE_COMPAT at the whole list.

Reply:
I think you can put sub-features of APX here if you do not have better candidates. `egpr,ndd,push2pop2,ppx,ccmp,cf,nf,zu`

Reply:
A bad news is that only X86_`zu` can be "used" here. all other sub-features of APX have already be "used."

Reply:
I don't understand this thread. What exactly needs to match what else?

Also, if it's important for each feature to be assigned a specific ABI-stable number for some reason, we should assign that number _directly_ in this macro, not indirectly via ordering...

Reply:
Could I just put this in place of these two lines for now?
```
X86_FEATURE       (DUMMYFEATURE1,           "__dummyfeature1")
X86_FEATURE       (DUMMYFEATURE2,          "__dummyfeature2")
```

Reply:
I've gone ahead and done that.

Reply:
Maybe remove here too https://github.com/llvm/llvm-project/blob/main/compiler-rt/lib/builtins/cpu_model/x86.c#L144